### DISCUSSION TITLE
./breeze not working for me

### DISCUSSION THREAD

**Discussion by mashirali** [apache/airflow#20918] [Q&A]
### Apache Airflow version

2.2.3 (latest released)

### What happened

i have run the ./breeze setup-autocomplete
then i ran source ~/.bash_completion.d/breeze-complete

then,
i ran       ./breeze --python 3.8 --backend mysql

the lines are from 270-277  in contributors quick start file

here is the error

![image](https://user-images.githubusercontent.com/57099739/149889117-43c1ce55-68e1-4fa2-a2b8-58839f7b67fc.png)

### What you expected to happen

Breeze should have worked

### How to reproduce

just follow the steps for airflow contributors quick start and error will come on line 277

### Operating System

windows 11

### Versions of Apache Airflow Providers

recently downloaded the main branch

### Deployment

Other Docker-based deployment

### Deployment details

A simple docker based deployment for making pipelines 

### Anything else

Its a start of my deployment process and this error came

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Reply by potiuk** [reply]
Do not use mounted files from windows. Run "git checkout' in Ubuntu filesystem. That should help.

**Reply by potiuk** [reply]
When you swtich and you have still problems, run `VERBOSE_COMMANDS="true" ./breeze -v` and share the output (ideally as gist - it will be huge).
